client/src/elements/ListItemBarColored.js

```diff
@@ -1,15 +1,15 @@
 import React from 'react'
 import { Box, Center, VStack, HStack, Pressable, Text } from 'native-base'
 
-export default ({ text, subtext, icon, onPress }) => (
+export default ({ text, subtext, icon, onPress, props }) => (
```

Comment:
rename variable `props` since text, subtext, icon, onPress are also part of props object.